src/commands/generateProject.ts

```diff
@@ -96,7 +96,13 @@ export async function generateProject(clickedFileUri: vscode.Uri | undefined): P
     try {
       await generateRestClient(jarCommand);
     } catch (e) {
-      if (e.message.includes(SPEC_VALIDATION_EXCEPTION)) {
+      console.error(e);
+      // get error description returned from executing jar command
+      let err = e.message.trim().split(jarCommand);
+      err = err[1].trim().split("\n")[0];
```

Comment:
We might want to check the length of the split error message here just in case there is some scenario where the error that is thrown from `generateRestClient` does not contain the jar command in it. And just show a generic error message in that case.